### ISSUE TITLE
Feature Request - Configuration for the Android/iOS shortcut

### ISSUE THREAD

**Issue by PhilippRoessner** [ionic-team/vscode-extension#124]
I'd like to configure shortcut of starting the Android or iOS application. important would be to set the project and the configuration parameters.

**Comment by dtarnawsky** [comment]
At the moment you can run the app in iOS, Android or Web, then using the keyboard shortcut of `alt+R` will re-run for that same configuration. I'm guessing though you are after the ability to be customize this.

**Comment by PhilippRoessner** [comment]
@dtarnawsky Yeah, like I mentioned, it would be useful to customize the project and configuration parameters for the run command. Right now, the "main" project with the production environment will start. I'd like to configure that.

**Comment by mbeckenbach** [comment]
@dtarnawsky I agree to @PhilippRoessner. When working on Angular projects it's pretty important that you can set a configuration as it is usually used for different environments. All platform Run and Debug options should support to set the configuration parameter.

**Comment by mbeckenbach** [comment]
@PhilippRoessner 
I just discoverd that you actually can change the Run links to other configurations. It's just not really intutive. 
First you need to hover over the ... next to the build feature and choose a config. Then the Run features "remember" it. 

![image](https://github.com/ionic-team/vscode-extension/assets/6654388/aa3a40fd-d532-419d-bbc5-c9869884e85c)
